src/cli/cli.go

```diff
@@ -334,6 +334,9 @@ func receive(c *cli.Context) (err error) {
 		if crocOptions.SharedSecret == "" {
 			crocOptions.SharedSecret = rememberedOptions.SharedSecret
 		}
+		if !c.IsSet("pass") {
```

Comment:
I'm only not sure what's the difference to `GlobalIsSet`, I saw you used a regular `IsSet` for the `send` command, that's why I did the same

Reply:
Thanks @maximbaz. I made a mistake! Both of them should be `GlobalIsSet`. You can update your PR or I can just do it :)

Reply:
Updated 👍